esp8266/main.c

```diff
@@ -59,6 +60,10 @@ void soft_reset(void) {
 void user_init(void) {
     mp_reset();
     mp_hal_stdout_tx_str("\r\n");
+#if MICROPY_MODULE_FROZEN
+    mp_lexer_t *lex = mp_find_frozen_module("main", 4);
```

Comment:
This will need to go in mp_reset(), so that when you press CTRL-D (soft reset) the script is run again.